jsonb/2.0/_index.md

```diff
@@ -1,23 +1,22 @@
 ---
-title: "Jakarta JSON Binding Specification 2.0 (milestone)"
-date: 2020-06-08
+title: "Jakarta JSON Binding 2.0"
+date: 2020-08-12
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta JSON Binding defines a binding framework for converting Java(R) objects to and from JSON documents.
 
 * [Jakarta JSON Binding 2.0 Release Record]()
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta JSON Binding 2.0 Specification Document](./jsonb-spec-2.0-RC2.pdf) (PDF)
-* [Jakarta JSON Binding 2.0 Specification Document](./jsonb-spec-2.0-RC2.html) (HTML)
+* [Jakarta JSON Binding 2.0 Specification Document](./jsonb-spec-2.0.pdf) (PDF)
```

Comment:
Suggestion... It was suggested that we preface the specs with "jakarta-" to help highlight the overall project.  Not a firm requirement for Jakarta EE 9, but since there are other things to clean up...  Do this for both the pdf and html.

Reply:
done